whylog/config/__init__.py

```diff
@@ -30,14 +34,21 @@ def _load_rules(self):
     def _load_log_types(self):
         pass
 
+    def _get_parsers_name(self):
+        parsers_name = OrderedSet()
+        for parser in self._parsers:
+            parsers_name.add(parser)
```

Comment:
Why do we add `parser` to `parser_name`? I thought `parsers_name` is a container for parsers names, not whole parsers. Maybe rename it or write docstring about `config._parsers_name`?